src/Resources/config/routing/shop.yaml

```diff
@@ -0,0 +1,2 @@
+bitbag_sylius_banner_plugin_banners:
+    resource: "@BitBagSyliusBannerPlugin/Resources/config/routing/shop/*.yaml"
```

Comment:
sometimes loaded in separate files, sometimes in one (it is better to stick to one convention)